src/universal/components/NewMeeting.tsx

```diff
@@ -264,10 +261,9 @@ class NewMeeting extends Component<Props> {
 
   gotoPrev = () => {
     const {
-      viewer: {
-        team: {newMeeting}
-      }
+      viewer: {team}
     } = this.props
+    const {newMeeting} = team!
```

Comment:
What does `team!` mean?

Reply:
manually flag team as non-null. before, when requesting a document like `viewer { team(teamId) {id}}` the graphql server guaranteed it would return a non-null team. now, it says it'll return null if you don't have access to that team. if the graphql server expects team to be non-null & it is null, then it will return `{viewer: null}`. what we want is `{viewer: {team: null}}`.

Probably worth a PR to GraphQL, but that'd be a major breaking change & a lot of back & forth...